source/reference/minio-mc-admin/mc-admin-heal.rst

```diff
@@ -85,4 +85,12 @@ Healing Colors
 --------------
 
 Some versions of MinIO used a color key as a way to differentiate objects with different healing statuses.
-For details of this key, see the :ref:`Healing <minio-concepts-healing-colors>` concept page.
+
+.. versionchanged:: mc RELEASE.2024-11-17T19-35-25Z
+
+The color meaning has been updated.
```

Comment:
Does the info linked to in the original text need to be updated as well? Or maybe it can go away.

https://min.io/docs/minio/linux/operations/concepts/healing.html#healing-output-color-key

Reply:
Might as well make it gone away.

Reply:
yeeted